src/main/java/uk/gov/companieshouse/api/accounts/controller/AccountController.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.companieshouse.api.accounts.controller;
+
+import javax.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.companieshouse.api.accounts.model.rest.Account;
+import uk.gov.companieshouse.api.accounts.service.AccountService;
+
+@RestController
+public class AccountController {
+
+    @Autowired
+    private AccountService accountService;
+
+    @PostMapping(value = "/transactions/{transactionId}/accounts",
```

Comment:
There is current discussion about changing this to company-accounts instead of accounts to simplify migration to the new repo. Are you going to raise that change as a change request to update it once the decision is made?

Reply:
ummm, if its merged by then then yes i will raise a new pr